content/guides/models/track/log/customize-logging-axes.md

```diff
@@ -6,51 +6,72 @@ menu:
 title: Customize log axes
 ---
 
-Use `define_metric` to set a **custom x axis**.Custom x-axes are useful in contexts where you need to log to different time steps in the past during training, asynchronously. For example, this can be useful in RL where you may track the per-episode reward and a per-step reward.
+Set a custom x-axis when you log metrics to W&B. By default, W&B logs metrics as a function of *steps*. A step corresponds to the number of times you call `wandb.log()`. 
```

Comment:
Not sure about "as a function of steps". Is it better to. just say that the X-axis defaults to step?